addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -56,6 +56,7 @@ export class MassMailingHtmlField extends HtmlField {
         return {
             ...super.wysiwygOptions,
             onIframeUpdated: () => this.onIframeUpdated(),
+            onWysiwygChange: () => this.commitChanges(),
```

Comment:
About the commit message: What does "upsing" mean ? Also, there is typo in "therfore".